src/LodeRunner.API/src/Controllers/HealthzController.cs

```diff
@@ -53,7 +54,7 @@ public HealthzController(ILogger<HealthzController> logger, ILogger<CosmosHealth
             Summary = "Healthz Check (simple)",
             Description = "Returns a text/plain health status (pass, warn or fail)",
             OperationId = "RunHealthzAsync")]
-        public async Task<IActionResult> RunHealthzAsync([FromServices] IClientStatusService clientStatusService, [FromServices] CancellationTokenSource cancellationTokenSource, [FromServices] Config config)
+        public async Task<IActionResult> RunHealthzAsync([FromServices] ClientStatusService clientStatusService, [FromServices] CancellationTokenSource cancellationTokenSource, [FromServices] Config config)
```

Comment:
Generally, we want the interface.  What is the reason for using the concrete class here?

Reply:
@gortegaMS Could you provide some context?

Reply:
Discussed it will be replacing concrete class with BaseService @jofultz

Reply:
Refactoring Interfaces in https://github.com/retaildevcrews/wcnp/issues/577.

Reply:
@alfredoihernandez

Reply:
done